src/components/Table.tsx

```
@@ -0,0 +1,71 @@
+
+
+
+
+interface TableProps {
+	data: [];
+	tableHead?: string[];
+	children: any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)